src/components/NavBar.jsx

```diff
@@ -0,0 +1,72 @@
+import { useState } from 'react';
+import NavLinks from './NavLinks';
+
+export function NavBar({ user, lists, listPath }) {
+	const [isNavOpen, setIsNavOpen] = useState(false);
+
+	return (
+		<>
+			{/* DESKTOP MENU */}
+			<nav className="h-16 hidden lg:flex flex-row shadow-md justify-between items-center bg-offWhite text-darkPurple">
+				<div className="h-full flex items-center pl-12">
+					<h1 className="font-amiri text-2xl font-bold">Smart Shopping List</h1>
+				</div>
+				{!!user && (
+					<div className="h-full flex flex-row items-center">
+						<NavLinks
+							lists={lists}
+							listPath={listPath}
+							setIsNavOpen={setIsNavOpen}
+						/>
+					</div>
+				)}
+			</nav>
+
+			<nav className="h-12 w-full flex flex-row justify-between lg:hidden relative shadow-md bg-offWhite">
+				{/* MOBILE-MENU */}
+				<div className="h-full flex items-center pl-12">
+					<h1 className="font-amiri text-2xl font-bold">Smart Shopping List</h1>
+				</div>
+				{!!user && (
+					<div className="flex items-center h-full pr-4">
+						{/* HAMBURGER-ICON */}
+						<button
```

Comment:
Can we use this icon here to add visual consistency with the rest of icons: https://fontawesome.com/icons/bars?f=classic&s=solid ?